auth0/README.md

```diff
@@ -0,0 +1,58 @@
+# Agent Check: auth0
+
+## Overview
+
+Use the Datadog-Auth0 integration to view and analyze your log events from Auth0.
+
+## Setup
+
+All configuration happens on the [Auth0 Dashboard][2]. Find details on how to set it up in the [Auth0 Documentation][3]. Datadog automatically creates a dashboard when it detects Auth0 logs.
+
+ 
+### Configuration
+
+1. Log in to the [Auth0 Dashboard][2].
+2. Navigate to **Logs** > **Streams**.
+3. Click **+ Create Stream**.
+4. Select Datadog and enter a unique name for your new Datadog Event Stream.
+5. On the next screen, provide the following settings for your Datadog Event Stream:
+
+
+    | Setting     	   | Description                                                |
+    | ---------------- | ---------------------------------------------------------- |
+    | `API Key`        | Enter your [Datadog API key][4]. 							|
+    | `Region` 		   | If you are in the Datadog EU site (app.datadoghq.eu), the Region should be `EU`, otherwise it should be `GLOBAL`   |
+
+	
+6. Click Save.
+7. You're done! Now when Auth0 writes the next tenant log, you'll receive a copy of that log event in Datadog with the source and service set to `auth0`.
+
+### Validation
+
+View logs in Datadog:
+
+1. Navigate to **Logs** > **Livetail**.
+2. See Auth0 logs by setting `service:auth0`.
+
+## Data Collected
+
+### Metrics
```

Comment:
we can add a heading with "### Logs" with some details on the kind of data these logs have. It doesn't need too much detail but perhaps either classes of logs being sent or important data types and a link to all data types https://auth0.com/docs/logs/references/log-event-type-codes

Reply:
Added `### Logs` section